### PR TITLE
Fix config provider ordering.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Fix config provider ordering, as suggested by @Colengms

### CODE REVIEW

File: Extension/src/LanguageServer/client.ts [hunk 1]
```diff
@@ -1856,9 +1856,11 @@ export class DefaultClient implements Client {
         if (!currentProvider.isReady) {
             return;
         }
+
         await Promise.all([
```
Comment (fearthecowboy) [inline Extension/src/LanguageServer/client.ts:1860]: Does `clearCustomConfigurations` need to complete before `handleRemoveAllCodeAnalysisProblems` ?

if not, then I'm ok with this.

if it does, just await them individually

**Review by fearthecowboy — APPROVED** [review]

File: Extension/src/LanguageServer/client.ts [hunk 2]
```diff
@@ -1856,9 +1856,11 @@ export class DefaultClient implements Client {
...
             this.clearCustomConfigurations(),
-            this.handleRemoveAllCodeAnalysisProblems(),
+            this.handleRemoveAllCodeAnalysisProblems()]);
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/client.ts:1862]: Is the formatting okay? The formatter doesn't make any changes.

**Review by Colengms — APPROVED** [review]

File: Extension/src/LanguageServer/client.ts (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts:1860]: The order doesn't matter between those two calls.

File: Extension/src/LanguageServer/client.ts (hunk 2 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts:1862]: Formatting seems okay to me...I don't see any existing code using the previous formatting.

**Pull request merged**